spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Not yet.  I wanted to get all these checked in so I can actually get to
work, and so that next time it's more clear what I've done (since there's
something like 27 files atm).  Thanks

On Fri, Sep 18, 2015 at 3:31 PM, Derek Kniffin <notifications@github.com>
wrote:

> In spec/models/user_spec.rb
> <https://github.com/dacur/bookstore/pull/1#discussion_r39892424>:
>
> > @@ -0,0 +1,5 @@
> > +require 'rails_helper'
> > +
> > +RSpec.describe User, type: :model do
> > +  pending "add some examples to (or delete) #{__FILE__}"
>
> No tests?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dacur/bookstore/pull/1/files#r39892424>.
>